model/assertion.go

```diff
@@ -25,11 +25,12 @@ import (
 // Assertion represents an expression in a section of the model.
 // For example: r = sub, obj, act
 type Assertion struct {
-	Key    string
-	Value  string
-	Tokens []string
-	Policy [][]string
-	RM     rbac.RoleManager
+	Key       string
+	Value     string
+	Tokens    []string
+	Policy    [][]string
+	PolicyMap map[string]int
+	RM        rbac.RoleManager
```

Comment:
Could you add `sync.RWMutex` to the Assertion?

Reply:
ok, I will add it

Reply:
I still think this mutex has some duplicates with SyncedEnforce

Reply:
I see, we can new a  PR to improve it.